tests/acceleration.py

```diff
@@ -60,6 +60,30 @@ def can_improve(e, context, assumptions : Exp = ETRUE, pool : Pool = RUNTIME_POO
     return found_improvement
 
 class TestAccelerationRules(unittest.TestCase):
+    def test_flatmap_sum(self):
+        allow_big_sets.value = True
```

Comment:
I approve of changes that cover different option values! However, this line might introduce a test ordering dependency. To safely change the value of an option for just the duration of one test, you can use [`common.save_property`](https://github.com/CozySynthesizer/cozy/blob/1faf4e73ade6c2e4f6f8585328f33b85ab763cf8/cozy/common.py#L432). For example, the synthesis tests use it to temporarily disable acceleration rules: https://github.com/CozySynthesizer/cozy/blob/3bd8036593b0b86ac731fdd288d4f546be62b49b/tests/synthesis.py#L226-L227